src/blockchain/assets.md

```diff
@@ -0,0 +1,83 @@
+# Assets
+
+Iroha has been built with few underlying assumptions about what the assets
+need to be.
+
+The assets can be **fungible** (every £1 is exactly the same as every other
+£1) or **non-fungible** (a £1 bill signed by the Queen of Hearts is not the
+same as a £1 bill signed by the King of Spades).
+
+The assets can also be **mintable** (you can make more of them) and
+**non-mintable** (you can only specify their initial quantity in the
+[genesis block](/guide/configure/genesis.md)).
```

Comment:
I guess the quantity is specified upon declaring (registering) the asset, which can happen not only in the genesis block

Reply:
I am saving all the comments that are not about the structure itself for future tasks, will revisit them once we get to updating the relevant docs.